x-pack/plugins/actions/server/builtin_action_types/webhook.ts

```diff
@@ -34,10 +34,20 @@ const ConfigSchema = schema.object(configSchemaProps);
 type ActionTypeConfigType = TypeOf<typeof ConfigSchema>;
 
 // secrets definition
-type ActionTypeSecretsType = TypeOf<typeof SecretsSchema>;
-const SecretsSchema = schema.object({
-  user: schema.string(),
-  password: schema.string(),
+export type ActionTypeSecretsType = TypeOf<typeof SecretsSchema>;
+const secretSchemaProps = {
+  user: schema.maybe(schema.string()),
```

Comment:
I believe we need to use `schema.nullable()` here, as we wouldn't handle the case of "deleting" a user/password when editing an alert - they'd be set as `undefined` when the alert is updated, and so left untouched in ES.

Reply:
updated, thanks!